de/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: news_post
+title: "Ruby 2.3.0-preview1 veröffentlicht"
+author: "naruse"
+translator: "Marvin Gülker"
+date: 2015-11-11 14:00:00 +0000
+lang: de
+---
+
+Wir freuen uns, die Veröffentlichung von Ruby 2.3.0-preview1
+bekanntgeben zu können.
+
+Ruby 2.3.0-preview1 ist die erste Vorschau auf Ruby 2.3.0, in die
+viele neue Features und Verbesserungen eingeflossen sind.
+
+So wird etwa das [Pragma für eingefrorene
+String-Literale](https://bugs.ruby-lang.org/issues/11473)
+eingeführt. Bereits seit Ruby 2.1 handelt es sich bei `"str".freeze`
+um eine auf besseren Speicherverbrauch optimierte Operation, mit Ruby
+2.3 wird nunmehr ein neuer magischer Kommentar und eine
+Kommandozeilenoption eingeführt, die alle String-Literale im Quellcode
+betrifft. Zu Debugging-Zwecken können Sie mithilfe der Option
+`--enable-frozen-string-literal-debug` feststellen, an welcher Stelle
+das Objekt erstellt wird, welches den Fehler `"can't modify frozen
+String"` auslöst.
+
+Daneben wird der [Safe Navigation
+Operator](https://bugs.ruby-lang.org/issues/11537) eingeführt, wie er
+bereits aus C#, Groovy und Swift bekannt ist. Er erleichtert die
+Behandlung von `nil` als `obj.&foo`. Außerdem wurden `Array#dig` und
+`Hash#dig` hinzugefügt.
+
+[did_you_mean.gem wird
```

Comment:
Ich meine, ich habe bisher in deutschen Büchern nur "das Gem" gelesen (habe aber z.B. Flanagan/Matsumoto als Beleg hier nicht zur Hand...).

Übrigens: "Juwel" -> "das Gem"; und "die Gem" wäre natürlich auch denkbar ("Bibliothek") :smile:

Richtlinien sind mir nicht bekannt; "das" ist wohl die neutralste Version...